lib/vssolution.hpp

```diff
@@ -38,12 +38,15 @@ class VisualizationSceneSolution : public VisualizationSceneScalarData
 
    gl3::GlDrawable lcurve_buf;
    gl3::GlDrawable line_buf;
-   gl3::GlDrawable bdr_buf;
+   gl3::GlDrawable bdr_buf1;
+   gl3::GlDrawable bdr_buf2;
```

Comment:
Why use two boundary buffers? Just Call `PrepareBoundary()` when toggling `drawbdr` to 1 or 2.